src/mnode/rpc/masternode.cpp

```diff
@@ -540,22 +554,27 @@ UniValue masternode(const UniValue& params, bool fHelp)
     if (strCommand == "make-conf") {
         if (params.size() != 6 && params.size() != 8)
             throw JSONRPCError(RPC_INVALID_PARAMETER,
-                               R"("masternode make-conf "alias" "mnAddress:port" "extAddress:port" "extP2P:port" "passphrase" "txid" "index"\n"
-                               "Create masternode configuration in JSON format:\n"
-                               "This will 1) generate MasterNode Private Key (mnPrivKey) and 2) generate and register MasterNode PastelID (extKey)\n"
-                               "If collateral txid and index are not provided, it will search for the first available non-locked outpoint with the correct amount (1000000 PSL)\n"
-                               "\nArguments:\n"
-                               "    "alias"             (string) (required) Local alias (name) of Master Node\n"
-                               "    "mnAddress:port"    (string) (required) The address and port of the Master Node's cNode\n"
-                               "    "extAddress:port"   (string) (required) The address and port of the Master Node's Storage Layer\n"
-                               "    "extP2P:port"       (string) (required) The address and port of the Master Node's Kademlia point\n"
-                               "    "passphrase"        (string) (required) passphrase for new PastelID\n"
-                               "    "txid"              (string) (optional) id of transaction with the collateral amount\n"
-                               "    "index"             (numeric) (optional) index in the transaction with the collateral amount\n"
-                               "\nCreate masternode configuration\n")" +
-                                   HelpExampleCli("masternode make-conf",
-                                                  R"("myMN" "127.0.0.1:9933" "127.0.0.1:4444" "127.0.0.1:5545" "bc1c5243284272dbb22c301a549d112e8bc9bc454b5ff50b1e5f7959d6b56726" 4)") +
-                                   "\nAs json rpc\n" + HelpExampleRpc("masternode make-conf", R"(""myMN" "127.0.0.1:9933" "127.0.0.1:4444" "127.0.0.1:5545" "bc1c5243284272dbb22c301a549d112e8bc9bc454b5ff50b1e5f7959d6b56726" 4")")
+                R"("masternode make-conf "alias" "mnAddress:port" "extAddress:port" "extP2P:port" "passphrase" "txid" "index"
```

Comment:
Shift all lines left to remove whitespace at the beggining

Reply:
done thanks